applications/serial_lte_modem/doc/DFU_AT_commands.rst

```diff
@@ -159,12 +159,15 @@ Response syntax
 
 ::
 
-  XDFUSIZE: <file_size>,<download_size>
+  XDFUSIZE: <file_size>,<download_size>,<crc32_checksum>
 
 * The ``<file_size>`` is an integer.
   It indicates the size of the DFU image file.
 * The ``<download_size>`` is an integer.
   It indicates the size that has been downloaded so far.
+* The ``<crc32_checksum>`` is an 32bit unsigned integer.
+  It indicates the IEEE CRC32 of image content that has been downloaded.
+  This crc32 can be used to check integrity when DFU with MCUBOOT.
```

Comment:
```suggestion
* The ``<crc32_checksum>`` is a 32-bit unsigned integer.
  It indicates the IEEE CRC32 of the image content that has been downloaded.
  You can use this CRC32 to check data integrity when you use MCUboot to perform a DFU.
```

Reply:
Should be more detailed. The CRC checksum should be checked right after #XDFURUN has transferred the downloaded data image from nRF91 to nRF5x, before configuring MCUBoot to use the new image.